modules/osdk-cli-reference-new.adoc

```diff
@@ -22,6 +22,9 @@ The `operator-sdk new` command creates a new Operator application and generates
 |`--api-version`
 |CRD `APIVersion` in the format `$GROUP_NAME/$VERSION`, for example `app.example.com/v1alpha1`. Used with `ansible` or `helm` types.
 
+|`--crd-version`
+|CRD version to generate (e.g., `v1`). Default setting is `v1beta1`.
+
 |`--generate-playbook`
 |Generate an Ansible playbook skeleton. Used with `ansible` type.
 
```

Comment:
In v0.19.x, we removed support in the `new` subcommand for Go projects. In line 53, we should remove the references to `--type=go`

Reply:
Per our discussion on Slack, I'll leave the `new` subcommand in place for Ansible and Helm projects at this time. Thanks for reviewing, Joe!